HW1/P2/P2.txt

```diff
@@ -1 +1,9 @@
 # Your discussion here
+The graph for part a is scaled by 1e7, which is why it
+looks relatively clustered. In fact, the partitioning is quite poor:
+some partitions did 0 or close to 0 work, while other partitions did
+huge amounts of work.  The graph for part b on the other hand is tightly
+clustered around a small range of values just above 200,000. Thus we see that
+the solution of a random repartition via PartitionBy balances work *extremely* well.
```

Comment:
I think *extremely* might be a bit of an over-exaggeration :)
I'm sure if we really wanted to, we could hardcode some configuration of workers to fully optimize for solving this problem, but I do agree with you that given the simplicity of the solution and the low cost, this solution works relatively well!